setup.py

```diff
@@ -50,8 +52,9 @@ def read(fname):
         'lxml',
         'oktaauth',
         'python-keystoneclient',
+        'six>=1.10.0',
     ],
-    python_requires='>=2.7, !=3.*',
+    python_requires='>=2.7',
```

Comment:
Have you verified that this is compatible with both python2.7 and python3.6?

Reply:
Tested on python2.7, didn't get python3.6 env.